src/ZoneServer/Skills/Handlers/Swordsman/PommelBeat.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Melia.Shared.Data.Database;
+using Melia.Shared.L10N;
+using Melia.Shared.Game.Const;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Combat;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.Skills.SplashAreas;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+using static Melia.Zone.Skills.SkillUseFunctions;
+using Melia.Zone.World.Actors.Monsters;
+
+namespace Melia.Zone.Skills.Handlers.Swordsman
+{
+	/// <summary>
+	/// Handler for the Swordsman skill Pommel Beat.
+	/// </summary>
+	[SkillHandler(SkillId.Swordman_PommelBeat)]
+	public class PommelBeat : IGroundSkillHandler
+	{
+		private const float StunDamageMultiplier = 3f; // 300% damage if target is stunned
+		private const float BaseIgnoreDefRate = 0.01f; // 1% ignore defense
+		private const float IgnoreDefRatePerLevel = 0.005f; // 0.5% ignore defense per level
+
+		/// <summary>
+		/// Handles skill, damaging targets.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="originPos"></param>
+		/// <param name="farPos"></param>
+		public void Handle(Skill skill, ICombatEntity caster, Position originPos, Position farPos, ICombatEntity target)
+		{
+			if (!caster.TrySpendSp(skill))
+			{
+				caster.ServerMessage(Localization.Get("Not enough SP."));
+				return;
+			}
+
+			skill.IncreaseOverheat();
+			caster.TurnTowards(farPos);
+			caster.SetAttackState(true);
+
+			var splashParam = skill.GetSplashParameters(caster, originPos, farPos, length: 50, width: 25, angle: 0);
+			var splashArea = skill.GetSplashArea(SplashType.Square, splashParam);
+
+			Send.ZC_SKILL_READY(caster, skill, originPos, farPos);
+			Send.ZC_SKILL_MELEE_GROUND(caster, skill, farPos, null);
+
+			this.Attack(skill, caster, splashArea);
+		}
+
+		/// <summary>
+		/// Executes the actual attack after a delay.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="splashArea"></param>
+		private async void Attack(Skill skill, ICombatEntity caster, ISplashArea splashArea)
+		{
+			var hitDelay = TimeSpan.FromMilliseconds(300);
+			var damageDelay = TimeSpan.FromMilliseconds(150);
+			var skillHitDelay = TimeSpan.Zero;
+
+			await Task.Delay(hitDelay);
+
+			var targets = caster.Map.GetAttackableEntitiesIn(caster, splashArea);
+			var hits = new List<SkillHitInfo>();	
+
+			foreach (var target in targets.LimitBySDR(caster, skill))
+			{
+				SkillModifier modifier = new SkillModifier();
+
+				if (target.IsBuffActive(BuffId.Stun))
+				{
+					modifier.DamageMultiplier = StunDamageMultiplier;
+				}
+				var targetSize = SizeType.M; // non-mobs are always size M
```

Comment:
While I see where you're going with this comment, it's another case of a comment that doesn't contribute much. You're setting M as the default and change it for mobs. That tells us that M is the intended default, and that's really all there is to it. All non-mobs are M as a result. The consequence of wanting to clarify this would mean that you'd have to do this every time you set M as the default somewhere.

Reply:
Essentially I mainly wanted to make it clear that this wasn't simply an oversight where this skill had only been considered for PVE, but if you prefer I can remove the comment.